frontend/src/app/modules/result-selection/components/result-selection-job-group/result-selection-job-group.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="card form-horizontal">
+  <h2>{{ 'frontend.de.iteratec.osm.resultSelection.jobGroup.title' | translate }}</h2>
+  <div class="form-group">
+    <select multiple class="form-control" id='job-groups-selection'>
+      <ng-container *ngIf="!isEmpty; else elseBlock" >
+          <option  *ngFor="let jobGroup of filteredJobGroups" [ngValue]="jobGroup.name" >
```

Comment:
```suggestion
          <option  *ngFor="let jobGroup of filteredJobGroups" [ngValue]="jobGroup.id" >
```
Please use the id as value.